packages/server/lib/browsers/cri-client.ts

```diff
@@ -169,98 +169,96 @@ export const create = Bluebird.method((target: websocketUrl, onAsynchronousError
       })
 
       enqueuedCommands = []
-    })
-    .catch((err) => {
+    } catch (err) {
       onAsynchronousError(errors.get('CDP_COULD_NOT_RECONNECT', err))
-    })
+    }
   }
 
-  const connect = () => {
+  const connect = async () => {
     cri?.close()
 
     debug('connecting %o', { target })
 
-    return chromeRemoteInterface({
+    const newCri = await chromeRemoteInterface({
```

Comment:
```suggestion
    cri = await chromeRemoteInterface({
```

Reply:
Good call! Addressed in https://github.com/cypress-io/cypress/pull/20825/commits/14c43b15c9b0c540c6657f2c5e19ebd0d46dcf12